dependencies.txt

```diff
@@ -5,9 +5,8 @@ shapely
 networkx
 owslib
 map2model
-loopprojectfile==0.1.3
+loopprojectfile==0.1.4
 beartype
 gdal==3.8.4
-hjson
 pytest
```

Comment:
Do we want to have "testing" "dev" and "release" requirements?

Reply:
There might not be a big difference between the different requirements for map2loop, and we'd have to maintain more files. Is there a specific advantage in doing this that you're thinking of?

Reply:
Using pyproject it is possible to have specific requirements for testing/development, I'm not sure how to do it using conda though. It isn't a big deal though

Reply:
I think it's a good idea for a new feature, will add this to the map2loop project, but perhaps we'll leave it out of this release.